client/src/components/Feed/Deals/Deals.js

```diff
@@ -82,18 +87,36 @@ class Venues extends Component {
         </Modal>
         <div className="venues-content">
           <CryptoRankings />
-          <div id="right" className="row column">
+
+          <div id="right" className="grid">
             {this.state.deals.map(deal => (
-              <div key={deal.id} className="col-sm-4 deal">
-
-                  <div className="card">
-                    <div className="card-body">
-                      <img
-                        className="card-img-top"
-                        src={deal.deal_image}
-                        alt="deal"
-                      />
-                      <h5 className="card-title">{deal.deal_name}</h5>
+              <div key={deal.id} className="deal">
+                <Link to={`/feed/deals/${deal.deal_name}`} style={{ textDecoration: 'none', color: "black" }} >
```

Comment:
@simonnguyen3054 I really like how you are doing the routing! 
Does everything inside the `<Link>` tags become the "link"? In other words can you click on any element inside the Link tags on the page and be taken to the new route?